app/data/compose/docker-compose-watcher-mobymask-v2.yml

```diff
@@ -1,6 +1,25 @@
 version: '3.2'
 
 services:
+  laconicd:
```

Comment:
This should come from the fixturenet-laconicd "pod". Is there a reason it has to be repeated here?